db/migrate/20220228000520_add_index_to_users_email.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToUsersEmail < ActiveRecord::Migration[6.1]
+  def change
+    add_index :users, :email, unique: true
```

Comment:
we have a unique constraint on username as well, so this should also have a unique index.

Reply:
Yeah that definitely makes sense, will make sure to insert that!